docs/api/index.md

```diff
@@ -9,7 +9,7 @@ nav_order: 5
 |:----------------------------------------------------------------------------------|
 | <span class="fs-5"> [Python API Docs](./python/api_summary.html){: .btn } </span>  | 
 | <span class="fs-5"> [Java API Docs](./java/index.html){: .btn} </span>   | 
-| <span class="fs-5"> [Objective C Docs](./objectivec/index.html){: .btn} </span> |
+| <span class="fs-5"> [Objective-C Docs](./objectivec/index.html){: .btn} </span> |
```

Comment:
This is pulled from the title of this page: https://edgchen1.github.io/onnxruntime/docs/api/objectivec-api.html so you should fix the md on that page to change the link text

Reply:
Sorry, not sure what you're suggesting that I fix here:
https://github.com/microsoft/onnxruntime/blob/9fa90d762b07a81394f5534f21fc474920966b7c/docs/api/objectivec-api.md
Could you elaborate?

This change updates "Objective C" to "Objective-C" for consistency.

Reply:
Ah, I misread the change.